Source/Particles/Collision/BinaryCollision/Coulomb/UpdateMomentumPerezElastic.H

```diff
@@ -102,86 +105,87 @@ void UpdateMomentumPerezElastic (
     T_PR const g2s = ( T_PR(1.0) - vcDv2*inv_c2 )*gc*g2;
 
-    // Compute s
-    T_PR s = 0;
+    // Compute variant relative velocity in cm frame
+    T_PR const muRst = g1s*m1*g2s*m2/(g1s*m1 + g2s*m2);
+    T_PR const vrelst = p1sm/muRst;
+
+    // Compute invariant relative velocity (frame independent)
+    T_PR const denom = T_PR(1.0) + p1sm*p1sm/(m1*g1s*m2*g2s)*inv_c2; // (1.0 - v1s*v2s/c^2)
+    T_PR const vrelst_invar = vrelst/denom;
+
+    // Compute s12
+    T_PR s12 = 0;
     if (p1sm > std::numeric_limits<T_PR>::min()) {
 
-        // s is non-zero (i.e. particles scatter) only if the relative
-        // motion between particles is not negligible (p1sm non-zero)
+        // Writing b0 in a form that is directly analagous to the well-known non-relativistic form.
+        // See Eq. 3.3.2 in Principles of Plasma Discharges and Material Processing by
+        // M. A. Lieberman and A. J. Lichtenberg.
+        // Note that b0 on Eq. 22 of Perez POP 19 (2012) is bmin = b0/2,
+        // Note: there is a typo in Eq 22 of Perez, the last square is incorrect!
+        // See the SMILEI documentation: https://smileipic.github.io/Smilei/Understand/collisions.html
+        // and https://github.com/ECP-WarpX/WarpX/files/3799803/main.pdf from GitHub #429
+        T_PR const b0 = amrex::Math::abs(q1*q2) /
+                        (T_PR(2.0)*MathConst::pi*PhysConst::ep0*muRst*vrelst*vrelst_invar);
+
 
         // Compute the Coulomb log lnLmd first
         T_PR lnLmd;
         if ( L > T_PR(0.0) ) { lnLmd = L; }
         else
         {
-            // Compute b0 according to eq (22) from Perez et al., Phys.Plasmas.19.083104 (2012)
-            // Note: there is a typo in the equation, the last square is incorrect!
-            // See the SMILEI documentation: https://smileipic.github.io/Smilei/Understand/collisions.html
-            // and https://github.com/ECP-WarpX/WarpX/files/3799803/main.pdf from GitHub #429
-            T_PR const b0 = amrex::Math::abs(q1*q2) * inv_c2 /
-                (T_PR(4.0)*MathConst::pi*PhysConst::ep0) * gc/mass_g *
-                ( m1*g1s*m2*g2s/(p1sm*p1sm*inv_c2) + T_PR(1.0) );
-
-            // Compute the minimal impact parameter
-            constexpr T_PR hbar_pi = static_cast<T_PR>(PhysConst::hbar*MathConst::pi);
-            const T_PR bmin = amrex::max(hbar_pi/p1sm, b0);
+
+            // Compute the minimum impact parameter from quantum: bqm = lDB/(4*pi) = hbar/(2*p1sm)
+            // See NRL formulary. Also see "An introduction to the physics of the Coulomb logarithm,
+            // with emphasis on quantum-mechanical effects", J. Plasma Phys. vol. 85 (2019). by J.A. Krommes.
+            // Note: The formula in Perez 2012 and in Lee and More 1984 uses h rather than
+            // hbar for bqm. If this is used, then the transition energy where bmin goes from classical
+            // to quantum is only 2.5 eV for electrons; compared to 100 eV when using hbar.
```

Comment:
Given that we cite the above paper, I think that this is not needed.
```suggestion
```

Reply:
I felt that since the formulation here largely follows Perez 2012, that it may be useful to make explicit comments where things differ. But if you feel it is not needed then I will take it out.

Reply:
I am ok to leave it, explicit is good where things differ.